src/utils.ts

```diff
@@ -380,6 +380,16 @@ export const setMaxStalePeriod = async (
         normalTimelock,
         maxStalePeriodInSeconds,
       );
+
+      if (underlyingAsset.address === "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB") {
```

Comment:
We could define a constant variable for this value, and use it here and in the file `src/vip-framework/checks/checkIsolatedPoolsComptrollers.ts`